app/src/androidTest/java/com/sdp/movemeet/view/chat/ChatActivityTest.java

```diff
@@ -56,6 +57,7 @@
 
     @Test
     public void chatActivityTest() {
+        ChatActivity.enableNav = false;
```

Comment:
What does this line do ? We can't see the navigation bar from the chat ?

Reply:
Kepler found this way to allow tests more easily, basically, in the classes we have this variable "enableNav" which is visible for the tests and if it is true, we instantiate the navigation class and therefore we create the menu. In the tests, we set it to false and like that, we don't instantiate the navigation class, so there is not even a menu for the tests because it is not necessary and would need to call other firebase dependencies